R/layout.R

```diff
@@ -954,10 +961,15 @@ layout_nicely <- function(graph, dim = 2, ...) {
     }
 
     args$graph <- graph
+
+    if (is_forest(graph) && vcount(graph) <= 30) {
+      return(align_layout(graph, do.call(layout_as_tree, args)))
+    }
+
     args$dim <- dim
 
     if (vcount(graph) < 1000) {
-      do.call(layout_with_fr, args)
+      align_layout(graph, do.call(layout_with_fr, args))
```

Comment:
Suggestion: Do not use alignment for the tree layout, which is already perfectly aligned.

Alignment makes sense for organic layouts such as `fr` and `kk`, but not for trees.